mlir/lib/Dialect/LLVMIR/Transforms/DIScopeForLLVMFuncOp.cpp

```diff
@@ -86,6 +86,10 @@ namespace {
 /// Add a debug info scope to LLVMFuncOp that are missing it.
 struct DIScopeForLLVMFuncOp
     : public LLVM::impl::DIScopeForLLVMFuncOpBase<DIScopeForLLVMFuncOp> {
+  DIScopeForLLVMFuncOp() = default;
+  DIScopeForLLVMFuncOp(const mlir::LLVM::DIScopeForLLVMFuncOpOptions &options)
```

Comment:
```suggestion
  explicit DIScopeForLLVMFuncOp(const mlir::LLVM::DIScopeForLLVMFuncOpOptions &options)
```

Except you want implicit conversion, ofc.

Reply:
Thanks, using `using Base::Base` instead.